source/js/components/distance-filter.js

```diff
@@ -0,0 +1,42 @@
+import results from './results.js';
+import ajaxData from './ajax.js';
+
+const distanceFilter = () => {
+
+    const rangeEl = document.querySelector('.js-range');
+    const filterActiveDistanceEl = document.querySelector('.js-active-distance');
+    let doctor =  results();
```

Comment:
[fix] Consider some better naming conventions here. This should also be a `const`.